### PR TITLE
created HTML & Css layout of Works section

### PR DESCRIPTION
**Pull request by davidobi**
- Added Linters setup
- Created the CSS of the works section using CSS Grid and Flexbox alongside other CSS syntax
- The HTML was basic. Images were exported from Figma.
- .gitignore was added to exclude node_modules
- Edited and updated readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>David's Portfolio - Mobile</title>
+</head>
+<body>
+    <section class="works">
```
Comment (elmar8287) [inline index.html:11]: - [ ] Please, consider adding the previous sections: toolbar, headline - I mean you should continue building your portfolio webpage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        <div class="grid-container">
+            <div class="card card1">
+                <img class="screenshot" src="./pics/homepage.png" alt="">
+                
+                <h1 class="main-heading"> Tonic </h1>
+
```
Comment (elmar8287) [inline index.html:17]: - [ ] Please, consider that the `h1` tag could be used one time on your webpage. All other headings should be `hx`, aka h2, h3, h4, etc.
- [ ] Please, fix it for all extra `h1` tags in your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev">Back End Dev</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev year">2015</p>
+                    </li>
+                </ul>
```
Comment (elmar8287) [inline index.html:30]: - [ ] Please, consider for each this `ul` in your code the alignment to left. In your version, it is in the center.
>Your version
Screen Shot 2022-02-11 at 18 12 06

> Figma design
Screen Shot 2022-02-11 at 18 12 37

File: index.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                <p class="paragraph">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
+                </p>
+                <ul class="language-button">
+                    <li><button type="button" class="l-button">html</button></li>
+                    <li><button type="button" class="l-button">css</button></li>
+                    <li><button type="button" class="l-button">javaScript</button></li>
+                </ul>
+                <button type="button" class="see-project">See Project</button>
```
Comment (elmar8287) [inline index.html:68]: - [ ] Please, consider for each "See Project" button the Figma button styling: Active, Hover and Pressed (see design)
Screen Shot 2022-02-11 at 18 18 30

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @davidobi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  Check the comments under the review.
- [ ]  Please, consider that you have to add the "About myself" section also, as mentioned in project requreremtns. (consider style it as Figma template)

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (davidobi) [reply index.html:11]: Well noted It has been implemented

File: index.html (hunk 2 above)
Comment (davidobi) [reply index.html:17]: Well noted thanks. It has been implemented.

File: index.html (hunk 3 above)
Comment (davidobi) [reply index.html:30]: Well noted. I have edited the Ul element and pushed it to the left.

File: index.html (hunk 4 above)
Comment (davidobi) [reply index.html:68]: Thanks for this awesome reminder. I have added the hover and pressed buttons where applicable throughout the project. @elmar8287 Please review. The updates have been pushed to the about me, work section repo. Please review.

Comment (elmar8287) [reply index.html:68]: @davidobi Perfect! 👍 Please, if you are finished with all suggestions, then submit your PR for the next review. Happy coding! 🥇

**Pull request merged**